Sources/SwiftInspectorVisitors/PropertySyntaxVisitor.swift

```diff
@@ -109,8 +109,8 @@ public final class PropertySyntaxVisitor: SyntaxVisitor {
 
 // MARK: - PropertyInfo
 
-public struct PropertyInfo: Hashable, CustomDebugStringConvertible {
-  public struct Modifier: Hashable, OptionSet {
+public struct PropertyInfo: Codable, Hashable, CustomDebugStringConvertible {
```

Comment:
one of these days we might want to make this struct `Equatable` but... we don't need to today.